packages/bot/src/components/journal.jsx

```diff
@@ -23,7 +23,7 @@ const FormatMessage = ({
     const key_words = ['Bought', 'Sold', 'Profit amount', 'Loss amount'];
```

Comment:
```suggestion
     if (typeof message !== 'string') {
         return message;
     }
    const key_words = ['Bought', 'Sold', 'Profit amount', 'Loss amount'];
```

Reply:
We should still format the output to a string before outputting it to the user.

Reply:
@aaron-binary well if it is undefind it will cause error when you do `message.split` a few lines down. i will check for it in store so if it is undefind i wont add it to list